Yoti.Auth.Sandbox/Profile/Request/YotiTokenRequestBuilder.cs

```diff
@@ -216,10 +216,12 @@ public YotiTokenRequestBuilder WithDocumentDetails(string value)
 
         public YotiTokenRequestBuilder WithDocumentDetails(string value, List<SandboxAnchor> anchors)
         {
+#pragma warning disable 0618
             SandboxAttribute sandboxAttribute = SandboxAttribute.Builder()
                     .WithName(UserProfile.DocumentDetailsAttribute)
                     .WithValue(value)
-                    .WithOptional(true)
+                    .WithOptional(true) //NOSONAR
```

Comment:
Should we remove this?
```suggestion
```

Reply:
Yes good point, should be removed. Will do in separate commit and remove the pragma's too